### ISSUE TITLE
sharing dialog not showing recievers after having done a lookup and closing the sidepanel

### ISSUE THREAD

**Issue by TheACF** [nextcloud/server#11856]
### Steps to reproduce
1.
Go to Share
2.
Type in the Field and wait for browsing the user directory
3.
Delete the letters you typed
4.
leave the menu with close
5.
Enter the sharing menue for the same folder again
6. 
Try again to type and wait for browsing directory

### Expected behaviour
Tell us what should happen
Some users from the directory should be listed
### Actual behaviour
Tell us what happens instead
In the second try - nothing happends...no user is listed and it seems that loading from userdirectory not happening
After Page Reload it is working again.
### Server configuration

**Operating system**:
Ubuntu 16.04
**Web server:**
Apache
**Database:**
MySQl
**PHP version:**
7.0
**Nextcloud version:** (see Nextcloud admin page)
14.0.3
**Updated from an older Nextcloud/ownCloud or fresh install:**
14.0.1 (the error was already on this version after upgrade from latest 13.)
**Where did you install Nextcloud from:**

**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here. 
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
<details>
<summary>Web server error log</summary>

```
Insert your webserver log here
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by violoncelloch** [comment]
reproduced
related to #10774 ?

**Comment by MorrisJobke** [comment]
Seems to be related to #10934

**Comment by skjnldsv** [comment]
Fixed with #15719
